### DISCUSSION TITLE
What to do with default.vim

### DISCUSSION THREAD

**Discussion by saccarosium** [vim/vim#15625] [Polls]
In #14853 #14883 we had discussed what to do with `default.vim` (to learn more `:h defaults.vim`).
To make the decision more informed, I ask the community what their opinion is and how this should be handled.

Please share this in your favorite vim community.

> [!Note]
> 
> If none of the following answers reflect your opinion. Please leave a comment with your choice or leave a :+1: in a comment that reflect your view.

**Comment by jmdevin** [comment]
This poll is pretty flawed - my preferred solution is not described well by any of the three options provided. If I don't vote, then anyone reading the results might be misled into believing that more of a consensus exists than there really is. So consider this my vote for option four: none of the above.

**Reply by saccarosium** [reply]
I didn't think about it to be honest. But you are making a good point.
Github doesn't make me change the poll, but if others are in your same position, I guess they can leave a comment as you did.

**Reply by ychin** [reply]
In addition to what @jmdevin said (which I agree with), *very* few of Vim users follow Vim that much to come here to vote. We should first accept that fact that a poll like this is going to be from a small subset of users who follow Vim development closely (and likely already know about defaults.vim well and therefore not going to the target audience).

**Comment by saccarosium** [comment]
Hey @habamax can you link this in the IRC?

**Comment by dlejay** [comment]
> This poll is pretty flawed

I voted `Never source default.vim` because I believe the defaults should be hard coded in Vim, not sourced from a `vim` file.

@saccarosium did I click the correct option reflecting my views ?
